source/dyaml/escapes.d

```diff
@@ -9,12 +9,12 @@ module dyaml.escapes;
 
 package:
 
-/// All YAML escapes.
-immutable dchar[] escapes = ['0', 'a', 'b', 't', '\t', 'n', 'v', 'f', 'r', 'e', ' ',
-                             '\"', '\\', 'N', '_', 'L', 'P'];
+import std.meta : AliasSeq;
+alias escapes = AliasSeq!('0', 'a', 'b', 't', '\t', 'n', 'v', 'f', 'r', 'e', ' ',
```

Comment:
what's the point of using AliasSeq vs a static immutable array  ?

Reply:
this is for internal use only, and it was only used as a parameter for canFind. that was changed to among, which doesn't handle arrays.

Reply:
ok